src/Migrations/M032MigrateCampaignCoversToP4ColumnsBlock.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+declare(strict_types=1);
+
+namespace P4\MasterTheme\Migrations;
+
+use P4\MasterTheme\MigrationRecord;
+use P4\MasterTheme\MigrationScript;
+use P4\MasterTheme\BlockReportSearch\BlockSearch;
+use WP_Block_Parser;
+
+/**
+ * Migrate Campaign covers block to Planet4 columns block.
+ */
+class M032MigrateCampaignCoversToP4ColumnsBlock extends MigrationScript
+{
+    private const BLOCK_NAME = 'planet4-blocks/covers';
+    private const POST_TYPES = [ 'page', 'post', 'action', 'campaign' ];
+    private const COVER_TYPE = 'campaign';
+    private const OLD_COVER_TYPES = [
+        '1' => 'take-action',
+        '2' => 'campaign',
+        '3' => 'content',
+    ];
```

Comment:
Some of these constants can be reused in other files (for example, [here](https://github.com/greenpeace/planet4-master-theme/pull/2363/files#diff-882ae83a59536852c35295e4efeeaf80e277d1f512cb5c10d2143e9fbad46c76R17)).
Do you think it is worth moving them to a separate file to avoid duplicate content?